crates/nu-command/tests/commands/lines.rs

```diff
@@ -61,3 +61,11 @@ fn lines_mixed_line_endings() {
 
     assert_eq!(actual.out, "3");
 }
+
+#[cfg(not(windows))]
+#[test]
+fn lines_on_error() {
+    let actual = nu!("open . | lines");
```

Comment:
In general I find tests using the filesystem outside the playground a bit sus, but I can also see the value of a direct minimal reproduction (as it will alert to a semantic change in `open` as bycatch)

Reply:
yeah, i went for no playground because the test does not even require the use of any file or directory :yum: